extensions/exposition/source/HTTP/Server.ts

```diff
@@ -71,18 +60,42 @@ export class Server extends Connector {
   protected override async close (): Promise<void> {
     const stopped = promex()
 
-    this.server?.close(stopped.callback)
+    this.server.close(stopped.callback)
 
-    await stopped
+    console.info('HTTP Server stopped accepting new connections.')
 
-    this.server = undefined
+    await stopped
```

Comment:
See above, this code will be easier to understand using standard `once(this.server, 'close')`

Reply:
I've updated `listen`, but I'm not sure if the same approach applicable here, since `.close()` **does** accepts a callback.

Reply:
I believe it's applicable to `close` as the only circumstances when `.close` callback can be called with an error is _it will be called with an Error as its only argument if the server was not open when it was closed_. I'm not 100% sure what is the lifecycle of those calls in Toa - both `open` and `close` are protected methods and I'm assuming we are in full control of execution flow and so that error should never happen.